cmd/client/cmd/login_cluster.go

```diff
@@ -2,12 +2,20 @@ package cmd
 
 import (
 	"fmt"
+	"os"
+
+	context "golang.org/x/net/context"
 
-	"github.com/go-openapi/strfmt"
 	"github.com/howeyc/gopass"
+	"github.com/luizalabs/teresa-api/cmd/client/connection"
+	"github.com/luizalabs/teresa-api/pkg/client"
 	"github.com/spf13/cobra"
+
+	userpb "github.com/luizalabs/teresa-api/pkg/protobuf"
 )
```

Comment:
You're right `protobuf` is the common package for all protobuf, but, I just don't like `pb` as package name on big code bases 😂